src/app/login/login.component.html

```diff
@@ -28,6 +28,8 @@ <h1 class="text-secondary" style="font-size: 4rem; font-weight: bold;">
               placeholder="Password"
             />
             <p class="errorText">{{ errors.password }}</p>
+            <!-- CAPTCHA -->
+            <re-captcha (resolved)="resolved($event)" siteKey="6Lcj5tsUAAAAAMQvCio4kLVCed4KmvcbkhSCOfp1"></re-captcha>
```

Comment:
Again, I didn't make it pop up on failed attempts yet cause i wasn't sure if we were gonna do it based on a counter locally or user based failed attempts